File: Data Structures & Algorithms/two-integer-sum/submission-1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        var dict = new Dictionary<int, int>();
        
        for (var i = 0; i < nums.Length; i++) {
            var difference = target - nums[i];

            if (dict.ContainsKey(difference)) {
                return new int[] {dict[difference], i};
            }

            dict[nums[i]] = i;
        }

        return null;
    }
}

File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        hashmap = {}

        for num in nums:
            if num in hashmap:
                hashmap[num] += 1
            else:
                hashmap[num] = 1

            if hashmap[num] > (len(nums) / 2):
                return num
        